solr/core/src/java/org/apache/solr/jersey/SubResponseAccumulatingJerseyResponse.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.solr.jersey;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Represents API responses composed of the responses of various sub-requests.
+ *
+ * <p>Many Solr APIs, particularly those historically reliant on overseer processing, return a
+ * response to the user that is composed in large part of the responses from all sub-requests made
+ * during the APIs execution. (e.g. the collection-deletion response itself contains the responses
+ * from the 'UNLOAD' call send to each core.) This class encapsulates those responses as possible.
+ */
+public class SubResponseAccumulatingJerseyResponse extends SolrJerseyResponse {
+
+  @JsonProperty("requestid")
+  public String requestId;
```

Comment:
Isn't the requestId here the same as "rid" which is only used in logging?  But maybe not because it's a response JSON property?

Reply:
I _think_ that Solr spits out 'requestid' here because that's the name of the parameter that [v1 REQUESTSTATUS](https://solr.apache.org/guide/6_6/collections-api.html#CollectionsAPI-requeststatus) has taken, historically.

Definitely something worth cleaning up at some point.  (Personally my preference would be for something more explicit, like `asynchronous-request-id`, but admittedly that's probably too verbose for most people's tastes.)  I'd be onboard with any option that helped requestid be less overloaded.

That said - definitely beyond the scope of this ticket to make any field naming changes, unfortunately.  Changing the response format would impact v1 APIs as well, have backcompat implications, etc.

Reply:
For these types of feedback, what's the best way foreward?   Should we create a JIRA for each one of these and make it be related to the current migration related JIRA, (in this case SOLR-16372)?   If so, do you want to do them as the person moving the current issue forward, or me to do it as "the kibitzing person coming up with ideas" person?  I don't want to lose these ideas improvements ;-)